test/services-view.spec.ts

```diff
@@ -90,7 +98,68 @@ describe('servicesView', () => {
     expect(jQuery(element).find('.services-item-name').length).toBe(1);
   });
 
+  it("should categorize items correctly when main category has 'tags'", () => {
+    // remove any dynamically added 'Other' categories
+    removeOtherCategories(Constants.SERVICE_CATALOG_CATEGORIES);
```

Comment:
we shouldn't be directly modifying the Constants.SERVICE_CATALOG_CATEGORIES from source or tests, we should be cloning that object first